.github/workflows/artifacts-download-trigger-build.yml

```
@@ -0,0 +1,198 @@
+name: artifacts-download-trigger Build & Test
+
+on:
+  push:
+    branches: [ main, 'feature/*' ]
+    paths:
+      - 'artifacts-download-trigger/**'
+  pull_request:
+    branches: [ main, 'feature/*' ]
+    paths:
+      - 'artifacts-download-trigger/**'
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '12'
+      - name: Build with ncc
+        run: |
+          cd artifacts-download-trigger
+          npm install -g yarn
+          yarn install
+          yarn run build
+      - name: Archive dist
+        uses: actions/upload-artifact@v2
+        with:
+          name: build
+          path: artifacts-download-trigger/dist
+
+  echo-1-test:
+    needs: [build]
+    runs-on: ubuntu-latest
+    name: "echo-1-test [trigger|by workflow name]"
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+      - name: Download dist
+        uses: actions/download-artifact@v2
+        with:
+          name: build
+          path: artifacts-download-trigger/dist
+      - name: Invoke echo 1 workflow using this action (do not wait for completion)
+        uses: ./artifacts-download-trigger
+        with:
+          workflow: artifacts-download-trigger Message Echo 1
+          gh_token: ${{ secrets.BOT_PAT_TOKEN }}
```

```suggestion
          gh_token: ${{ secrets.GREENBONE_BOT_TOKEN }}
```

We want to use a token that actually exists ...